src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx

```diff
@@ -11,7 +11,11 @@ export interface AnnotationActivityLinkProps {
 }
 
 const AnnotationActivityLink = ({ id, message, onClick = noop }: AnnotationActivityLinkProps): JSX.Element => {
-    const handleClick = () => {
+    const handleClick = (event: React.SyntheticEvent) => {
+        event.preventDefault();
+        event.stopPropagation();
+        event.nativeEvent.stopImmediatePropagation(); // Prevents document event handlers from executing
```

Comment:
Which document listeners are we preventing and why?

Reply:
We're preventing the native click event from propagating because that would trigger the `useEventOutside` hook logic in the box-annotations `RegionList` and deselect the annotation

Reply:
Can we add a comment with that information?